bench/dune

```diff
@@ -0,0 +1,18 @@
+(executable
+ (name bench)
+ (public_name bench)
+ (package ocamlformat-bench)
+ (libraries bechamel bechamel-js ocamlformat_lib stdio))
+
+(rule
+ (alias runbench)
+ (package ocamlformat-bench)
+ (deps source_bench.ml)
```

Comment:
I don't suppose there is any way to write `../test/passing/tests/source.ml` here instead? Or a symlink? Or it would also help me appease the bots if test code like `bench_source.ml` was in a separate directory like `bench/test` or similar.

Reply:
done